### DISCUSSION TITLE
can we get a windows version also?

### DISCUSSION THREAD

**Discussion by TecnikOfficial** [Divested-Mobile/hypatia#31] [Ideas]
this app is very helpful for my old android low end device but can u make app like this for windows ... i tried clamwin but it doesnot scans my idm downloads automatically ... i tried proprietary software but they consume lot of resources

**Comment by SkewedZeppelin** [comment]
Just use Windows Defender if you're already running Windows.

It is highly effective especially when adjusted and controlled folder access is enabled.

**Comment by TecnikOfficial** [comment]
no i dont want realtime protection just want to scan my idm downloads like hypatia scans downloaded apks
